src/content/governance/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Ethereum Governance
+description: An introduction to how decisions about Ethereum are made.
+lang: en
+sidebar: true
+---
+
+# Introduction to Ethereum Governance {#introduction}
+
+_If no one owns Ethereum, how are decisions about past and future changes to Ethereum made? Ethereum Governance refers to the process that allows such decisions to be made_
```

Comment:
Should "Ethereum Governance" be capitalized as if it's a proper noun?

Reply:
Probably not